csv/src/fs2/data/csv/CsvRowDecoder.scala

```diff
@@ -55,6 +125,11 @@ object CsvRowDecoder extends ExportedCsvRowDecoders {
       }
     }
 
+  implicit def CsvRowDecoderSemigroupK[Header]: SemigroupK[CsvRowDecoder[*, Header]] =
```

Comment:
Minor consistency thing: Maybe combine the instance of `MonadError` and `SemigroupK` into one as for the other decoder types?